app/src/test/kotlin/com/intive/picover/parties/viewmodel/PartyDetailsViewModelTest.kt

```diff
@@ -0,0 +1,29 @@
+package com.intive.picover.parties.viewmodel
+
+import androidx.lifecycle.SavedStateHandle
+import com.intive.picover.parties.model.Party
+import com.intive.picover.parties.repository.MockedPartiesRepository
+import com.intive.picover.parties.state.PartyDetailsState
+import io.kotest.core.spec.style.ShouldSpec
+import io.kotest.matchers.shouldBe
+import io.mockk.every
+import io.mockk.mockk
+
+class PartyDetailsViewModelTest : ShouldSpec(
+	{
+
+		val partyId = 2
+		val party: Party = mockk()
+		val repository: MockedPartiesRepository = mockk()
+		val savedStateHandle: SavedStateHandle = mockk {
+			every { get<Int>(any()) } returns partyId
```

Comment:
You could use real object here, please check the constructor of `SavedStateHandle`. 

Note that test is useless, you can change the line in `PartyDetailsViewModel`: 
from `savedStateHandle["partyId"]` to `savedStateHandle["chupachups"]`
and test would be green :)

Please watch out with using `any()`, you shouldn't avoid, it simplifies the code but but you have to keep at least one unit test, which would detect such regression